index.html

```diff
@@ -163,63 +162,6 @@ <h3>
         </p>
       </aside>
     </section>
-    <section>
-      <h3>
-        Permissions and user prompts
-      </h3>
-      <p>
-        The [[PERMISSIONS]] API provides a uniform way for websites to request
-        permissions from users and query which permissions they have.
-      </p>
-      <p>
-        A <a>user agent</a> can <dfn data-lt=
-        "deny wake lock|denies the wake lock">deny a wake lock</dfn> of a
-        particular <a>wake lock type</a> for a particular {{Document}} by any
-        implementation-specific reason, such as platform setting or user
-        preference.
-      </p>
-      <p>
-        It is RECOMMENDED that a user agent show some form of unobtrusive
-        notification that informs the user when a wake lock is active, as well
-        as provides the user with the means to [=screen wake lock permission
-        revocation algorithm|block=] the ongoing operation, or simply dismiss
-        the notification.
```

Comment:
This text has gotten lost in this revision.

Reply:
Oops! Good catch.

Reply:
#323 ended up doing some of this, but I guess this text needs to be merged into the existing paragraph in the security section.